### DISCUSSION TITLE
How to use PSIR reconstruction in the Gadgetron?

### DISCUSSION THREAD

**Discussion by adamyliu** [gadgetron/gadgetron#1191] [Q&A]
I have LGE data and want to use the PSIR reconstruction program in Gadgetron to process the data (mentioned in [Gadgetron: Open Source Image Reconstruction](https://cdn0.scrvt.com/39b415fb07de4d9656c7b516d8e2d907/1800000003913232/f165deac5e03/magnetom-flash-68_gadgetron-open-source-image-reconstruction_hansen_1800000003913232.pdf) paper), but I don't find the reconstruction code in the source code of gadgetron. I would like to know how to implement the PSIR reconstruction mentioned in the above paper, and what code files are relevant to it.

**Comment by hansenms** [comment]
@xueh2
